react/w-books/src/app/components/Button/index.tsx

```diff
@@ -10,11 +11,28 @@ interface ButtonProps {
   isSubmit?: boolean;
   isWidthAuto?: boolean;
   handleClick?: () => void;
+  isLink?: boolean;
+  to?: string;
 }
 
-export default function Button({ children, isSubmit, className, isFilled, isWidthAuto, handleClick }: ButtonProps) {
+export default function Button({ children, isSubmit, className, isFilled, isWidthAuto, handleClick, to, isLink }: ButtonProps) {
 
   return (
+    isLink ?
```

Comment:
I think the best thing is that you separate this functionality into 2 different components, Button and Link, since their purpose is different things